src/app.js

```diff
@@ -1 +1,83 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios from "axios";
+
+// performs 1 API call before storing the result in countriesPromise
+async function buildLocalCache() {
+    try {
+       return (await axios.get('https://restcountries.com/v3.1/all')).data;
+    } catch (e) {
+        console.log(e);
+    }
+}
+
+async function fetchCountry(requestedCountry) {
+    // receives the promise from the cache
+    const countryArray = await countriesPromise
```

Comment:
Is er een reden waarom je en promise function gebruikt? Ik dacht dat dat in principe niet nodig was bij het gebruik van try/catch. Maar goed, als het werkt dan werkt het natuurlijk.